content/docs/tasks/security/plugin-ca-cert/index.md

```diff
@@ -133,6 +142,42 @@ This requires you have `openssl` installed on your machine.
     /tmp/pod-cert-chain-workload.pem: OK
     {{< /text >}}
 
+## (Optional) Generating root and intermediate keys and certificates offline
+
+This section gives you instructions on how to generate the keys and certificates offline and apply to
+the Istio cluster.
+
+On an offline machine with good security (restricted access), follow the `Create the root pair` and the
+`Create the intermediate pair` sections in this [guide](https://jamielinux.com/docs/openssl-certificate-authority/)
+to create the keys and certificates for your root and intermediate CA.
+
+After the certificates and keys are created,
```

Comment:
use callout to specify where the roto and intermediate certificates are located.  For reference https://preliminary.istio.io/about/contribute/creating-and-editing-pages/#callouts